packages/lib-subject-viewers/.storybook/main.js

```diff
@@ -0,0 +1,16 @@
+const config = {
+  stories: ["../src/**/*.stories.js"],
+  addons: [
+    "@storybook/addon-a11y",
+    "@storybook/addon-essentials",
+    "storybook-react-i18next",
```

Comment:
```suggestion
```

`i18n` should wait for actual content and a separate PR.

Reply:
Removed